webapp-django/crashstats/crashstats/models.py

```diff
@@ -28,6 +31,7 @@
 import socorro.external.postgresql.product_build_types
 import socorro.external.postgresql.signature_first_date
 import socorro.external.postgresql.server_status
+import socorro.external.boto.crash_data
```

Comment:
Why do you import both this module and ``SimplifiedCrashData`` from that same module?